packages/components/src/custom-select-control-v2/test/index.tsx

```diff
@@ -100,7 +106,7 @@ describe( 'With Legacy Props', () => {
 		} );
 
 		it( 'Should keep current selection if dropdown is closed without changing selection', async () => {
-			render( <CustomSelect { ...legacyProps } /> );
```

Comment:
These were all pointing to the uncontrolled version of the component, even though it was part of a uncontrolled/controlled matrix.

Reply:
What a good find!